src/main/java/org/mule/module/json/internal/ValidateJsonSchemaOperation.java

```diff
@@ -90,11 +90,12 @@ public void stop() {
   public void validateSchema(@Summary("The schema location") @Path(type = FILE, acceptedFileExtensions = "json") String schema,
                              @Content InputStream content,
                              @NullSafe @Optional Collection<SchemaRedirect> schemaRedirects,
-                             @Optional(defaultValue = "CANONICAL") JsonSchemaDereferencingMode dereferencing) {
+                             @Optional(defaultValue = "CANONICAL") JsonSchemaDereferencingMode dereferencing,
+                             @Optional(defaultValue = "true") boolean allowDuplicateKeys) {
```

Comment:
missing description. The SDK will take the `@param` javadoc and turn it into documentation